ORA-00933: SQL command not properly ended

The error below occured while using Atlas.  We have Oracle Database 19c Enterprise Edition. 
The error is caused by the ending semicolon in this DELETE statement:

```sql
String deleteSql = String.format("DELETE FROM %s.cohort_inclusion WHERE cohort_definition_id = %d;", targetSchema, cohortDef.getId());
```
Why this error happens was explained here: https://stackoverflow.com/questions/28544688/why-i-obtain-this-sqlsyntaxerrorexception-ora-00933-sql-command-not-properly

> executeQuery() automatically adds a semicolon to a statement when executing it.

Removing the semicolon solved the problem in our installation: https://github.com/OHDSI/WebAPI/commit/a5035b7246f0ed37c85e79cb3572cc8bf38c8d65  but we didn't test it with other RDMS. 
 
```
2020-12-31 08:58:01.410 ERROR taskExecutor-4 org.ohdsi.webapi.cohortdefinition.GenerateCohortTasklet -  - StatementCallback; bad SQL grammar [DELETE FROM OMOP_COHORT.cohort_inclusion WHERE cohort_definition_id = 3;]; nested exception is java.sql.SQLSyntaxErrorException: ORA-00933: SQL command not properly ended

org.springframework.jdbc.BadSqlGrammarException: StatementCallback; bad SQL grammar [DELETE FROM OMOP_COHORT.cohort_inclusion WHERE cohort_definition_id = 3;]; nested exception is java.sql.SQLSyntaxErrorException: ORA-00933: SQL command not properly ended

	at org.springframework.jdbc.support.SQLExceptionSubclassTranslator.doTranslate(SQLExceptionSubclassTranslator.java:91)
	at org.springframework.jdbc.support.AbstractFallbackSQLExceptionTranslator.translate(AbstractFallbackSQLExceptionTranslator.java:73)
	at org.springframework.jdbc.support.AbstractFallbackSQLExceptionTranslator.translate(AbstractFallbackSQLExceptionTranslator.java:82)
	at org.springframework.jdbc.core.JdbcTemplate.execute(JdbcTemplate.java:418)
	at org.springframework.jdbc.core.JdbcTemplate.update(JdbcTemplate.java:543)
	at org.ohdsi.webapi.cohortdefinition.CohortGenerationUtils.insertInclusionRules(CohortGenerationUtils.java:34)
	at org.ohdsi.webapi.cohortdefinition.GenerateCohortTasklet.prepareQueries(GenerateCohortTasklet.java:80)
	at org.ohdsi.webapi.common.generation.CancelableTasklet.doTask(CancelableTasklet.java:33)
	at org.ohdsi.webapi.common.generation.CancelableTasklet.doTask(CancelableTasklet.java:17)
	at org.ohdsi.webapi.common.generation.TransactionalTasklet.lambda$execute$0(TransactionalTasklet.java:37)
	at org.springframework.transaction.support.TransactionTemplate.execute(TransactionTemplate.java:133)
	at org.ohdsi.webapi.common.generation.TransactionalTasklet.execute(TransactionalTasklet.java:37)
	at org.springframework.batch.core.step.tasklet.TaskletStep$ChunkTransactionCallback.doInTransaction(TaskletStep.java:406)
	at org.springframework.batch.core.step.tasklet.TaskletStep$ChunkTransactionCallback.doInTransaction(TaskletStep.java:330)
	at org.springframework.transaction.support.TransactionTemplate.execute(TransactionTemplate.java:133)
	at org.springframework.batch.core.step.tasklet.TaskletStep$2.doInChunkContext(TaskletStep.java:272)
	at org.springframework.batch.core.scope.context.StepContextRepeatCallback.doInIteration(StepContextRepeatCallback.java:81)
	at org.springframework.batch.repeat.support.RepeatTemplate.getNextResult(RepeatTemplate.java:374)
	at org.springframework.batch.repeat.support.RepeatTemplate.executeInternal(RepeatTemplate.java:215)
	at org.springframework.batch.repeat.support.RepeatTemplate.iterate(RepeatTemplate.java:144)
	at org.springframework.batch.core.step.tasklet.TaskletStep.doExecute(TaskletStep.java:257)
	at org.springframework.batch.core.step.AbstractStep.execute(AbstractStep.java:200)
	at org.springframework.batch.core.job.SimpleStepHandler.handleStep(SimpleStepHandler.java:148)
	at org.springframework.batch.core.job.AbstractJob.handleStep(AbstractJob.java:392)
	at org.springframework.batch.core.job.SimpleJob.doExecute(SimpleJob.java:135)
	at org.springframework.batch.core.job.AbstractJob.execute(AbstractJob.java:306)
	at org.springframework.batch.core.launch.support.SimpleJobLauncher$1.run(SimpleJobLauncher.java:135)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.sql.SQLSyntaxErrorException: ORA-00933: SQL command not properly ended

	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:494)
	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:446)
	at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:1054)
	at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:623)
	at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:252)
	at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:612)
	at oracle.jdbc.driver.T4CStatement.doOall8(T4CStatement.java:213)
	at oracle.jdbc.driver.T4CStatement.doOall8(T4CStatement.java:37)
	at oracle.jdbc.driver.T4CStatement.executeForRows(T4CStatement.java:896)
	at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1119)
	at oracle.jdbc.driver.OracleStatement.executeUpdateInternal(OracleStatement.java:1661)
	at oracle.jdbc.driver.OracleStatement.executeLargeUpdate(OracleStatement.java:1626)
	at oracle.jdbc.driver.OracleStatement.executeUpdate(OracleStatement.java:1613)
	at oracle.jdbc.driver.OracleStatementWrapper.executeUpdate(OracleStatementWrapper.java:282)
	at org.springframework.jdbc.core.JdbcTemplate$1UpdateStatementCallback.doInStatement(JdbcTemplate.java:531)
	at org.springframework.jdbc.core.JdbcTemplate$1UpdateStatementCallback.doInStatement(JdbcTemplate.java:528)
	at org.springframework.jdbc.core.JdbcTemplate.execute(JdbcTemplate.java:407)
	... 26 more
Caused by: Error : 933, Position : 71, Sql = DELETE FROM OMOP_COHORT.cohort_inclusion WHERE cohort_definition_id = 3;, OriginalSql = DELETE FROM OMOP_COHORT.cohort_inclusion WHERE cohort_definition_id = 3;, Error Msg = ORA-00933: SQL command not properly ended

	at oracle.jdbc.driver.T4CTTIoer11.processError(T4CTTIoer11.java:498)
	... 42 more
```

--- a/src/main/java/org/ohdsi/webapi/cohortdefinition/CohortGenerationUtils.java
+++ b/src/main/java/org/ohdsi/webapi/cohortdefinition/CohortGenerationUtils.java
@@ -1,9 +1,7 @@
 package org.ohdsi.webapi.cohortdefinition;
 
-import java.util.Arrays;
 import org.apache.commons.lang3.StringUtils;
 
-import org.ohdsi.circe.cohortdefinition.CohortExpression;
 import org.ohdsi.circe.cohortdefinition.CohortExpressionQueryBuilder;
 import org.ohdsi.circe.cohortdefinition.InclusionRule;
 import org.ohdsi.sql.SqlRender;
@@ -30,9 +28,9 @@ public class CohortGenerationUtils {
   public static void insertInclusionRules(CohortDefinition cohortDef, Source source, int designHash,
                                           String targetSchema, String sessionId, JdbcTemplate jdbcTemplate) {
     final String oracleTempSchema = SourceUtils.getTempQualifier(source);
-    String deleteSql = String.format("DELETE FROM %s.cohort_inclusion WHERE cohort_definition_id = %d;", targetSchema, cohortDef.getId());
+    String deleteSql = String.format("DELETE FROM %s.cohort_inclusion WHERE cohort_definition_id = %d", targetSchema, cohortDef.getId());
     String translatedDeleteSql = SqlTranslate.translateSql(deleteSql, source.getSourceDialect(), sessionId, oracleTempSchema);
-    Arrays.stream(SqlSplit.splitSql(translatedDeleteSql)).forEach(jdbcTemplate::execute);
+    jdbcTemplate.update(translatedDeleteSql);
 
     String insertSql = StringUtils.replace("INSERT INTO @target_schema.cohort_inclusion (cohort_definition_id, design_hash, rule_sequence, name, description) VALUES (?,?,?,?,?)", "@target_schema", targetSchema);
     String translatedInsertSql = SqlTranslate.translateSql(insertSql, source.getSourceDialect(), sessionId, oracleTempSchema);

--- a/src/main/java/org/ohdsi/webapi/cohortdefinition/CohortGenerationUtils.java
+++ b/src/main/java/org/ohdsi/webapi/cohortdefinition/CohortGenerationUtils.java
@@ -1,5 +1,6 @@
 package org.ohdsi.webapi.cohortdefinition;
 
+import java.util.Arrays;
 import org.apache.commons.lang3.StringUtils;
 
 import org.ohdsi.circe.cohortdefinition.CohortExpression;
@@ -31,7 +32,7 @@ public class CohortGenerationUtils {
     final String oracleTempSchema = SourceUtils.getTempQualifier(source);
     String deleteSql = String.format("DELETE FROM %s.cohort_inclusion WHERE cohort_definition_id = %d;", targetSchema, cohortDef.getId());
     String translatedDeleteSql = SqlTranslate.translateSql(deleteSql, source.getSourceDialect(), sessionId, oracleTempSchema);
-    jdbcTemplate.update(translatedDeleteSql);
+    Arrays.stream(SqlSplit.splitSql(translatedDeleteSql)).forEach(jdbcTemplate::execute);
 
     String insertSql = StringUtils.replace("INSERT INTO @target_schema.cohort_inclusion (cohort_definition_id, design_hash, rule_sequence, name, description) VALUES (?,?,?,?,?)", "@target_schema", targetSchema);
     String translatedInsertSql = SqlTranslate.translateSql(insertSql, source.getSourceDialect(), sessionId, oracleTempSchema);

